src/Resources/app/administration/src/module/swag-migration/snippet/de-DE.json

```
@@ -209,20 +209,20 @@
         },
         "resetChecksumsConfirmDialog": {
           "title": "Reset der Prüfsummen",
-          "content": "Wenn Du die sogenannten Prüfsummen zurücksetzt, werden bei der nächsten Migration alle Daten aus dem Quellsystem noch einmal berücksichtigt. Dies beinhaltet auch bereits migrierte, unveränderte Datensätze. <br><br><strong>Wichtig:</strong> Änderungen der Datensätze, die Du seit der letzten Migration in Shopware 6 vorgenommen hast, werden wieder überschrieben. Die Daten aus dem Quellsystem haben Vorrang.",
+          "content": "Durch das Zurücksetzen der Prüfsummen wird Ihr Quellsystem in einen Zustand zurückversetzt, der dem Zustand vor dem ersten Migrationsversuch ähnelt. Wenn Sie sich entscheiden, erneut eine Migration durchzuführen, werden alle Änderungen, die in der Zwischenzeit am neueren Shopware 6-Zielsystem vorgenommen wurden, möglicherweise überschrieben.",
           "contentHint": "Möchtest du die Prüfsummen zurücksetzen?",
           "abortButtonText": "Abbrechen",
           "confirmButtonText": "Ja, zurücksetzen"
         },
         "resetMigrationConfirmDialog": {
-            "title": "Die Migration abschließen",
-            "header": "Deine Shopdaten sind fertig migriert?",
-            "content": "Super! Schließe dann jetzt die Migration ab, um temporäre und ungenutzte Dateien aufzuräumen. <strong>Bitte beachte:</strong> Das Abschließen der Migration ist endgültig. Du kannst danach keine Migration mehr über Deine aktuelle Verbindung durchführen.",
-            "contentHint": "Möchtest Du die Migration jetzt abschließen?",
+            "title": "Migration archivieren",
+            "header": "Hast du deine Shopdaten fertig migriert?",
```

we should follow these guidlines: https://shopware.atlassian.net/wiki/spaces/SHB/pages/20512637156/Writing+Texting+Wording+-+Dos+and+Don+ts

```suggestion
            "header": "Hast Du Deine Shopdaten fertig migriert?",
```